go/internal/error_messages.go

```diff
@@ -0,0 +1,11 @@
+package internal
+
+const (
+	ACTION_JSON_MARSHAL          = "marshal json"
+	ACTION_JSON_UNMARSHAL        = "unmarshal json"
+	ACTION_HTTP_BUILD_REQUEST    = "build http request"
+	ACTION_HTTP_DO_REQUEST       = "perform request to the endpoint"
+	ACTION_HTTP_READ_BYTES       = "read response bytes"
+	ACTION_API_SUCCESSFUL_RESULT = "receive successful result from API"
```

Comment:
Why do we need this?

Reply:
These are the actions we put in the `FailedTo` function. I put them into constants for consistency across the SDK